### ISSUE TITLE
`layer` should be read from `@import`

### ISSUE THREAD

**Issue by bartveneman** [projectwallace/css-analyzer#326]
From [Adam Argyle's CodePen](https://cdpn.io/pen/debug/OJoYwgx):

```css

@import "https://unpkg.com/open-props" layer(base);
@import "https://unpkg.com/open-props/normalize.min.css" layer(base);
@import "https://unpkg.com/open-props/oklch-hues.min.css" layer(base);
@import "https://unpkg.com/open-props/gray-oklch.min.css" layer(base);
```

I'd expect `base` to be a layer name, but currently no layers are reported.

**Comment by bartveneman** [comment]
CSSTree 2.3.1 does not yet parse `layer()` in `@import` yet.

https://stackblitz.com/edit/node-28ttpj?file=index.js,package.json

**Comment by bartveneman** [comment]
Example `@import` parsing in Svelte: https://github.com/sveltejs/svelte/pull/9098/files using CSSTree's `fork()` syntax.

**Comment by bartveneman** [comment]
WIP as copied from the Svelte PR: https://stackblitz.com/edit/stackblitz-starters-umuh7n?file=index.js,csstree-fork.js,nodes%2Fcontainer-query.js,nodes%2Fquery-css-function.js,nodes%2Fquery-feature-range.js
